include/mega/node.h

```diff
@@ -347,8 +394,21 @@ struct MEGA_API LocalNode : public File
 
         // checked for missing attributes
         bool checked : 1;
+
+        // needs another syncdown after pending changes
+        unsigned syncdownTargetedAction: 2;
+
+        // at least one child has needsAnotherSyncdown set
+        unsigned syncupTargetedAction: 2;
+
     };
 
+    // set the syncupPartialAction for this, and parents
```

Comment:
Should this be "syncupTargetedAction"?

Reply:
yes thanks - will adjust.  renamed it during the PR construction :-)